assets/src/components/v2/elevator/closures.tsx

```diff
@@ -141,7 +142,7 @@ const UpcomingClosure = ({
 
   return (
     <div className="upcoming-closure">
-      <CalendarAlertIcon width={224} />
+      <CalenderAlertIcon height={249} width={249} />
```

Comment:
Should we also revert the size change here? It's 224 in the design and is non-square (though it's actually 224 in _height_ rather than width, which might be a mistake in the original code).

Reply:
I don't think it makes a huge difference.... I'll revert it to 224 height I think.